public/app/features/datasources/settings/DataSourceSettingsPage.tsx

```diff
@@ -38,55 +41,20 @@ export interface Props {
   updateDataSource: typeof updateDataSource;
   setIsDefault: typeof setIsDefault;
   dataSourceLoaded: typeof dataSourceLoaded;
+  initDataSourceSettings: typeof initDataSourceSettings;
+  testDataSource: typeof testDataSource;
   plugin?: GenericDataSourcePlugin;
```

Comment:
`plugin` prop was already a prop before the move, do you know why? Could be something I overlooked when we did the move to Redux.

Reply:
I looked and there's no where else where this component is used or where `plugin` is passed in as a prop.